tests/test_multiconxn.py

```diff
@@ -57,16 +55,14 @@ def test_meta(self):
     def test_read(self):
         c2 = fiona.open(self.shapefile_path, "r")
         f2 = next(iter(c2))
-        del f2['id']
-        assert self.f == f2
+        assert self.f.properties == f2.properties
```

Comment:
Is this an API breaking change? Does the comparison of features no longer work?

Reply:
Feature comparison restored.